### DISCUSSION TITLE
NextAuth with Adonis JWT

### DISCUSSION THREAD

**Discussion by kaisiant** [nextauthjs/next-auth#2615] [Help]
Hello, is it possible to use adonis and nextauth together in my nextjs application? Currently I am facing a flash of authenticated content even the user is not logged in.

**Comment by balazsorban44** [comment]
Have you checked if you don't cache something by accident? Do you have some code you could show? It's hard to help without more info.

**Reply by kaisiant** [reply]
Using provider credentials solved my problem.
